pkg/notes/maps/testdata/fullmap.yaml

```diff
@@ -0,0 +1,28 @@
+---
+pr:  123
+commit: 1a89038915fe77d73bf7c9cfa8f2ce123a464c82
+release-note:  
+  text: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+  author: kubernetes-ci-robot
+  areas:
+    - testarea
+  kinds:
+    - documentation
+  sigs:
+    - api-machinery
+  feature: true
+  action_required: false
+  release_version: v1.18.4
+  documentation:
+    - description: Test documentation
+      url: https://wkdkjdjkdkfj
+      type: kep
+datafields:
+  score: 5.2
```

Comment:
The score should be part of the `cve`

Reply:
You're right. I thought this was the ML quality score. 

The idea was to illustrate a different field. In this case the ML quality score. While cve data would be a whole struct this is only a float .

Reply:
Okay, I guess we do not have to add anything related to ML for now.

(I was thinking about being able to link multiple PRs together, so we could add something like `linkedPRs`.)

Reply:
I've added more (hypothetical) examples to the fullmap.yaml file and updated the cve field. Although right now i'm only adjusting how the data would fit in the map. Once we settle how we inject the new fields we can decide on the actual formats for the fields and how they would render.